Makefile

```diff
@@ -1,21 +1,30 @@
 export CURRENT_PATH := $(shell pwd)
 export CUSTOM_THEME_PATH=${CURRENT_PATH}/custom-theme
 
-export PORT=8080
-export SQL_PASSWORD=example
-export URL=http://localhost:8080
+export PORT ?= 8080
+export SQL_PASSWORD ?= example
+export URL ?= http://localhost:8080
 
-export SMTP_MAIL_FROM='web <web@mydomain.com>'
-export SMTP_MAIL_USER=user@example.com
-export SMTP_MAIL_PASS=1234
-export SMTP_MAIL_HOST=smtp.mail.com
-export SMTP_MAIL_PORT=25
+export SMTP_MAIL_FROM ?= 'web <web@mydomain.com>'
+export SMTP_MAIL_USER ?= user@example.com
+export SMTP_MAIL_PASS ?= 1234
+export SMTP_MAIL_HOST ?= smtp.mail.com
+export SMTP_MAIL_PORT ?= 25
+
+# if defined, use curl instead of git
+export USE_CURL :=
 
 dummy		    := $(shell touch artifacts)
 include ./artifacts
 
 ${CUSTOM_THEME_PATH}:
-	git clone https://github.com/datalab-mi/Casper.git custom-theme
+	if [ -n "${USE_CURL}" ]; then \
```

Comment:
why use curl instead of git clone ?

Reply:
if git not present on prod server, download all as simple http packages

Reply:
docker-deploy.sh is the root install script on the server 
Simple install as : 
`curl http://...docker-deploy.sh  | bash `